packages/core/fluxcd/Makefile

```diff
@@ -8,13 +8,11 @@ show:
 
 apply:
 	helm template -n $(NAMESPACE) $(NAME) . --no-hooks --dry-run=server $(API_VERSIONS_FLAGS) | kubectl apply -n $(NAMESPACE) -f-
+	kubectl apply -n $(NAMESPACE) -f flux-instance.yaml
```

Comment:
Helm implementing resources sorting, where CRDs are always on the top,

So I think we can simple move it under `packages/core/fluxcd/templates`

It will also provide the same behavior for `make show` and `make diff` commands